pkg/gardenlet/controller/shoot/shoot_control.go

```diff
@@ -379,7 +381,9 @@ func (r *shootReconciler) deleteShoot(ctx context.Context, logger logrus.FieldLo
 		return reconcile.Result{}, fmt.Errorf("failed to check for related Bastions: %w", err)
 	}
 	if hasBastions {
-		return reconcile.Result{}, errors.New("shoot has still Bastions")
+		hasBastionError := errors.New("shoot has still Bastions")
```

Comment:
For consistency with the above similar error:
```suggestion
		hasBastionErr := errors.New("shoot has still Bastions")
```

Reply:
✅